src/main/webapp/fundingSourceReport/FundingsSourceReport.js

```diff
@@ -5,7 +5,7 @@ import { spinnerService } from "../util/spinner-service";
 import { Spinner } from "../components/Spinner";
 import { TableComponent } from "../components/TableComponent";
 import { handleRedirectToProject, printData } from "../util/Utils";
-import { SORT_NAME_INDEX, STYLES } from "../util/FundingsSourceReportConstants";
+import { FUNDING_SORT_NAME_INDEX, STYLES_FUNDING_REPORTS } from "../util/ReportConstants";
```

Comment:
Not sure on the exact syntax, but I think if you import as, you can revert a bunch of changes below that don't add any real information.
```suggestion
import { FUNDING_SORT_NAME_INDEX, STYLES_FUNDING_REPORTS as STYLES} from "../util/ReportConstants";
```